hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -174,40 +123,39 @@ commitTx ::
   Party ->
   -- | A single UTxO to commit to the Head
   -- We currently limit committing one UTxO to the head because of size limitations.
-  Maybe (Api.TxIn, Api.TxOut Api.CtxUTxO Api.Era) ->
+  Maybe (TxIn, TxOut CtxUTxO Era) ->
   -- | The inital output (sent to each party) which should contain the PT and is
   -- locked by initial script
-  (TxIn StandardCrypto, PubKeyHash) ->
-  ValidatedTx Era
+  (TxIn, Hash PaymentKey) ->
```

Comment:
I think this should stay a `Plutus.PubKeyHash`. After all, it's what we had de-coded when `observeInitTx`?